5_databases/week2/hw-p3.sql

```diff
@@ -0,0 +1,40 @@
+use lesson2;
+
+--Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT * FROM user WHERE email LIKE '%@spotify.com';
```

Comment:
Here you should use the user_task many-to-many table to make relations, like you do below